umap/static/umap/locale/am_ET.js

```diff
@@ -287,7 +287,7 @@ var locale = {
     "The zoom and center have been setted.": "ዙሙ እና መሀከሉ ተወስነዋል",
     "To use if remote server doesn't allow cross domain (slower)": "ሪሞት ሰርቨሩ የዶሜይን ልውውጥን የማይፈቅድ ከሆነ ይህን ለመጠቀም (አዝጋሚ)",
     "To zoom": "ለማጉላት",
-    "Toggle edit mode (shift-click)": "Toggle edit mode (shift-click)",
+    "Toggle edit mode (Shift+Click)": "Toggle edit mode (Shift+Click)",
```

Comment:
Humm, any idea why we only have three changes in translations files, while we have more changes in the code base ?

Reply:
Ah, because only three strings have the shortcut embedded, other are on the form `L._("xxx") + " (shortcut)"`